README.md

```diff
@@ -52,7 +52,6 @@ write messages onto the queue:
 
 ```ruby
 message_data = {
-  "event_type" => "heartbeat",
   "data" => {
```

Comment:
Data seems almost unneeded anymore... but we should keep it as it helps us conform to the form of JsonApi we are looking for in other places. I think I could be beneficial to continue to conform to that even in the formatting of our messages. As a good start, `"data"` is a top level item in our JsonApi world.

[JsonApi.org](http://jsonapi.org/)
[Fastly fst-json_api](https://github.com/fastly/fst-json_api)
 
It might be good to think about what we want the body of our messages to conform to before we have to many.